src/RapidezServiceProvider.php

```diff
@@ -80,6 +84,10 @@ protected function bootRoutes(): self
             $this->loadRoutesFrom(__DIR__.'/../routes/api.php');
         }
 
+        FallbackRoutes::add(UrlRewriteController::class, 5);
+        FallbackRoutes::add(CmsPageController::class, 10);
+        FallbackRoutes::add(LegacyFallbackController::class, 10000000);
```

Comment:
A bit too obvious? 99, 999 of 9999 should also do the job.

Reply:
We want this ones priority to be lower than the default which is currently `99999`
Of course both of these numbers are completely arbitrary and could be whatever number as long as they are high end the legacy fallback is lower than the default.

I could put the default as `9999` and the fallback as `99999` so theres enough space before and after in case a fallback needs to have lower priority than is the default but higher than the legacy fallback